go.mod

```diff
@@ -1,31 +1,37 @@
 module github.com/stakater/IngressMonitorController/v2
 
-go 1.18
+go 1.21
```

Comment:
Were u able to downgrade deps which were causing this change? because we should be on 1.18 according to this https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.22.0/#go-gov3-and-hybrid-helm-hybridhelmv1-alpha-language-based-operators---add-support-for-go-118

Reply:
I did not not manage to make it work on 1.18, no :disappointed: Will that be a problem you think? All the tests are running and linting is fine :thinking:

Reply:
should not be problem, we generally follow operator sdk for changes like this. But if you are confident that nothing will break, we can continue with this. Linter shouldve catch packages breaking.